Packs/SysAid/Integrations/SysAid/SysAid.py

```diff
@@ -877,6 +1054,7 @@ def main() -> None:
         elif command == 'test-module':
             test_module(client, params)
         elif command in commands:
+            args['username'] = username
```

Comment:
Please move the command that gets the `username` to a different case and send it directly, so future commands will not have problems with such an argument.